concert/base.py

```diff
@@ -1180,7 +1185,7 @@ async def __ainit__(self):
         pass
 
 
-class Parameterizable(AsyncObject):
+class Parameterizable(AsyncObject, abc.ABC):
```

Comment:
@MarcusZuber we are 10000 % sure this and the ABCMeta doesn't break anything, yes?

Reply:
I found a few classes in ankaconcert that needed some fixes (because they were not done correctly). 
Otherwise, I use this since December without any other issues.

Reply:
OK, that pretty much convinces me.